runtime/compiler/p/codegen/J9CodeGenerator.cpp

```diff
@@ -351,19 +351,6 @@ bool J9::Power::CodeGenerator::suppressInliningOfRecognizedMethod(TR::Recognized
       return true;
       }
 
-   // Transactional Memory
-   if (self()->getSupportsTM())
-      {
-      if (method == TR::java_util_concurrent_ConcurrentHashMap_tmEnabled ||
-          method == TR::java_util_concurrent_ConcurrentHashMap_tmPut ||
-          method == TR::java_util_concurrent_ConcurrentLinkedQueue_tmOffer ||
```

Comment:
Need to keep CLQ methods around?  A little too much are removed here.

Reply:
This is a mistake. Thanks for catching that. I've generated a diff of the change before and after and looked at all the + and - lines and checked that I'm not making a similar mistake elsewhere. Fixed in 5e429ad